schema/me/consignments/submissions.js

```diff
@@ -0,0 +1,18 @@
+// @ts-check
+
+import { SubmissionType } from "./submission"
+import { GraphQLBoolean } from "graphql/type/scalars"
+
+export default {
+  type: SubmissionType,
+  description: "The current user's submissions",
+  args: {
+    completed: {
```

Comment:
Do you also want to stick the pagination params in here? (`page` and `size`)?

Reply:
Oh yeah, dang, this needs to be a connection